### PR TITLE
Add assembly version information #6276

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change addresses #6276 by introducing AssemblyInfo.cs files for each asmdef in the Assets folder tree. As part of adding the version information, product name and copyright fields have been populated.

As part of this change, the updateMRTKversion script has been modified to process these files as well.

### CODE REVIEW

File: Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/Importers/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Reflection;
+
+[assembly: AssemblyVersion("2.2.0.0")]
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/Importers/AssemblyInfo.cs]: Do we need to go out to 4 digits for these? We've typically only used 3 for MRTK.

File: scripts/packaging/createnugetpackages.ps1 [hunk 2]
```diff
@@ -137,15 +137,22 @@ try
...
+        # ===
+        # https://github.com/microsoft/MixedRealityToolkit-Unity/issues/6276 states that not having a version on the assemblies in our NuGet packages
+        # is an issue... commenting the localVersion update
+        # ===
+
         # To make debugging the MRTK NuGet packages locally much easier automatically create new packages with version 0.0.0 and then
         # restore them to the machine NuGet feed. To test changes to the packages developers can run this script and then change their
         # project to consume version 0.0.0 and restore. Because the package is in the machine global feed it will resolve properly.
-        $localVersion = '0.0.0'
+        # $localVersion = '0.0.0'
```
Comment (keveleigh) [inline scripts/packaging/createnugetpackages.ps1]: Remove?

File: Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/Importers/AssemblyInfo.cs (hunk 1 above)
Comment (davidkline-ms) [reply Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/Importers/AssemblyInfo.cs]: the file format uses 4.

Comment (davidkline-ms) [reply Assets/MixedRealityToolkit/Utilities/Gltf/Serialization/Importers/AssemblyInfo.cs]: i had visual studio create one and used the same format

File: scripts/packaging/createnugetpackages.ps1 (hunk 2 above)
Comment (davidkline-ms) [reply scripts/packaging/createnugetpackages.ps1]: i'd like to leave it in until we are sure we no longer need / want it

**Pull request merged**